### ISSUE TITLE
NVDA doesn't read the description text in the "Options for managing of volume control" dialog of Windows 7

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3149]
**Reported by k_kolev1985 on 2013-04-12 15:31**
In Windows 7, when you invoke the context menu (right-click with the mouse) for the "Volume control" icon in the system tray, there are several options in there. The last of them is something like "Options for managing volume control" (translated from bulgarian - I don't know how it is called in english). When you press Enter on that option, a dialog appears. NVDA doesn't read the text witch is located in the upper portion of that dialog.

Steps to reproduce:
1. Press WindowsKey+B to go to the system tray.
2. Go to the icon for volume control (usually labeled with the friendly name of the current audio device).
3. Press the "applications" key (or Shift+F10) to invoke the context menu.
4. Select the option "Options for managing volume control" and press Enter on it. The "Options for managing volume control" dialog appears.
5. Press NVDA+B to read the whole dialog. Alternatively, point the mouse cursor over the top portion of that dialog (with "mouse tracking" turned on).
Expected result: NVDA should read a textual description for what is this dialog.
Actual result: There is a text in that upper portion of the dialog, witch NVDA doesn't read.
Actually, when I point the mouse cursor over that "invisible" text, NVDA doesn't say even "unknown" - as if that is not real text, but a graphic of some sort. But I know that is text, because with the "Virtual revision" addon I can get to that text and read it.

**Comment by bhavyashah** [comment]
Since the English locale of Windows works properly with NVDA as reported in https://github.com/nvaccess/nvda/issues/3149#issuecomment-155306230, it seems possible that this is a problem in the Bulgarian Windows locale itself, something NVDA might have little control over. Triaging thoughts? @jcsteh

**Comment by Adriani90** [comment]
@derekriemer, @josephsl your thoughts are very  appreciated here. I can still reproduce it in Windows 10 1809 Update.

**Comment by josephsl** [comment]
Hi,

I cannot reproduce this on Windows 7 English VM I have. Also, I see no volume mixer options dialog as noted in Windows 10 Version 1809 or other earlier release VM's I work with.

Please try with NVDA 2018.4/2019.1 alpha and see if that makes any difference.

Thanks.

**Comment by k-kolev1985** [comment]
I'm the reporter of this bug.

I've done some more testing now and here are my results.

1. The bug occurs under Windows 7 if the standard "Windows 7" theme is used, but not if the "high-contrast black" theme is used, for example. Apparently the problem has something to do with how that problematic text is rendered, because it seams to be rendered differently (at least with different fonts) depending on the theme being used.
2. Also, the bug may be locale dependent as well, since according to previous comments, does not happen with english locale.
3. Under Windows 8.1, the bug occurs no matter if the standard or the high-contrast black Windows theme is being used, probably because under Windows 8 and newer, the text is rendered the same newer way in both standard and high-contrast Windows themes. Under Windows 7, the high-contrast themes are rendered with the older "classic" view of Windows, including with different fonts.
4. Under Windows 10 (version 1809), I can no longer see the option for managing volume control in that context menu. I think it has been removed altogether or redesigned and moved to a different place in Windows.

Based on all said above and in previous comments, I think it is time to close this issue, since I doubt that the NVDA developers can do something about it. And considering that Windows 7 and Windows 8.1 are no longer actively developed, I also doubt that the Windows developers will do something about it.

**Comment by ehollig** [comment]
@k-kolev1985 thank you for your comment in https://github.com/nvaccess/nvda/issues/3149#issuecomment-444786952. I am going to close this issue for reasons pointed out above.
